### PR TITLE
Linting Python Files

### PR DESCRIPTION
**Pull request by gruunday**

### CODE REVIEW

**Comment by butlerx** [comment]
please merge in origin master

**Comment by butlerx** [comment]
we use 100 char line length not 80

**Comment by gruunday** [comment]
Why not stick with pep8

File: useradm/useradm.py [hunk 1]
```diff
@@ -788,47 +815,59 @@ def search():
...
 
+
 def show_search_results(res, fd):
     """Actual routine to display search results on given output steam."""
 
     if res:
-        print('%-*s %-*s %-8s %-30s %-6s %-4s %s' % (rbconfig.maxlen_uname, 'username', rbconfig.maxlen_group, 'usertype', 'id', 'name', 'course', 'year', 'email'), file=fd)
-        print('%s %s %s %s %s %s %s' % ('-' * rbconfig.maxlen_uname, '-' * rbconfig.maxlen_group, '-' * 8, '-' * 30, '-' * 6, '-' * 4, '-' * 30), file=fd)
+        print('%-*s %-*s %-8s %-30s %-6s %-4s %s' % (rbconfig.maxlen_uname, 'username',
+                                                     rbconfig.maxlen_group, 'usertype', 'id', 'name', 'course', 'year', 'email'), file=fd)
```
Comment (butlerx) [inline useradm/useradm.py:864]: this wrapping looks wrong

File: useradm/useradm.py [hunk 2]
```diff
@@ -1795,7 +1895,8 @@ def get_usertype(usr):
 
     while 1:
         if interact:
-            usr.usertype = ask('Enter usertype', defans, hints = [i for i in rbconfig.usertypes_list if opt.mode != 'renew' or i in rbconfig.usertypes_paying])
+            usr.usertype = ask('Enter usertype', defans, hints=[
+                               i for i in rbconfig.usertypes_list if opt.mode != 'renew' or i in rbconfig.usertypes_paying])
```
Comment (butlerx) [inline useradm/useradm.py:1899]: this also looks wrong

**Comment by butlerx** [comment]
because we are using codeclimate for linting "codeclimate — no new or fixed issues"

File: useradm/useradm.py (hunk 1 above)
Comment (gruunday) [reply useradm/useradm.py:864]: It looks very readable? :P

**Review by gruunday — COMMENTED** [review]
Why would that matter? Stick with pep8 standards?

File: useradm/useradm.py (hunk 1 above)
Comment (gruunday) [reply useradm/useradm.py:864]: This is not wrong.

File: useradm/useradm.py (hunk 2 above)
Comment (gruunday) [reply useradm/useradm.py:1899]: This is also the rules. Not wrong

**Comment by butlerx** [comment]
it matters because codeclimate enforces our linting on pr. its what people will use to check they follow the style guide

File: useradm/useradm.py (hunk 1 above)
Comment (butlerx) [reply useradm/useradm.py:864]: this line is 138 char long would it bit be 
```
        print('%-*s %-*s %-8s %-30s %-6s %-4s %s' %
              (rbconfig.maxlen_uname,'username', rbconfig.maxlen_group, 'usertype', 'id', 'name',
               'course', 'year', 'email'), file=fd)
``` 
if youre using 80

File: useradm/useradm.py (hunk 2 above)
Comment (butlerx) [reply useradm/useradm.py:1899]: this line is also above 100 char

**Comment by gruunday** [comment]
Yes but why is pep8 not our style guide

**Comment by butlerx** [comment]
because we decided to use pylint and thats what sre intergrations are configured to check for

File: useradm/useradm.py (hunk 2 above)
Comment (VoyTechnology) [reply useradm/useradm.py:1899]: If we have an inline `for` this long, I believe it qualifies to be split

**Comment by gruunday** [comment]
We? Why? Pros, Cons?

File: useradm/useradm.py (hunk 2 above)
Comment (butlerx) [reply useradm/useradm.py:1899]: i more mean the line is too long

**Comment by butlerx** [comment]
i dont remember the pros and cons anymore :stuck_out_tongue: im forgetful.
it was discussed and agreed upon

**Comment by gruunday** [comment]
But who. I want to know what the majority of people on this project think is the best style guide to follow?

**Comment by CSmartt** [comment]
Anyone got a log of this discussion/agreement? Might clear this up and save us all having to read this back&forth.

**Comment by butlerx** [comment]
i didnt keep a list :P i know @VoyTechnology and @GoldenBadger were part of the discussion

**Comment by gruunday** [comment]
Does pylint not implement pep8 as default?

**Comment by GoldenBadger** [comment]
According to their website it does.

File: useradm/useradm.py (hunk 1 above)
Comment (gruunday) [reply useradm/useradm.py:864]: My bad my browser was in fact wrapping this for me. Yes that would be it

**Comment by butlerx** [comment]
:man_shrugging: all i know is its had us writing for 100  char

**Comment by butlerx** [comment]
https://gist.github.com/butlerx/a7cc82ccb5b40a4e3d02c87c26769a58#file-pylint-txt-L11 this is from default pylint

**Comment by gruunday** [comment]
Okay this seems to be the biggest difference. We will continue with pylint until such time that we find it does not suit us.

**Pull request closed without merging**